scilpy/utils/streamlines.py

```diff
@@ -63,8 +63,8 @@ def intersection(left, right):
     return {k: v for k, v in left.items() if k in right}
 
 
-def subtraction(left, right):
-    """Subtraction of two streamlines dict (see hash_streamlines)"""
+def difference(left, right):
```

Comment:
This function is used in `tractanalysis/reproducibility_measures.py:compute_bundle_adjacency_streamlines`